pkg/chartverifier/checks/charttesting.go

```diff
@@ -256,3 +290,32 @@ func installAndTestChartRelease(
 
 	return result
 }
+
+func setOCVersion(opts *CheckOptions, versioner Versioner) error {
```

Comment:
`opts` is being used only to obtain an AnnotationHolder in the context, so it could be changed to an AnnotationHolder value instead. I particularly don't like this domain of annotation of a report going all the way to a check that should be agnostic.

Reply:
Done.